src/govuk/components/footer/_index.scss

```diff
@@ -106,17 +106,35 @@
 
   .govuk-footer__copyright-logo {
     display: inline-block;
-    min-width: $govuk-footer-crest-image-width;
-    padding-top: ($govuk-footer-crest-image-height + govuk-spacing(2));
+    text-align: center;
+    white-space: nowrap;
+  }
+
+  .govuk-footer__copyright-logo:before {
```

Comment:
Rather than using padding we create a pseudo element that is before the text so we can avoid abusing padding-top and give a box the proper sizing, which apart from making this technique work feels a bit nicer.